network/service.go

```diff
@@ -182,12 +182,7 @@ func (s *Service) handleStream(stream libp2pnetwork.Stream) {
 	r := bufio.NewReader(stream)
 
 	for {
-		// read leb128 variable-length encoding
-		_, err := common.ReadByte(r)
-		if err != nil {
-			log.Error("Failed to read message encoding", "peer", peer, "err", err)
-			return // exit
-		}
+		// TODO: re-add leb128 variable-length encoding
```

Comment:
reopened #484

Reply:
pls add the issue number to the comment so it can be tracked back.